lib/puppet/pops/lookup/hiera_config.rb

```diff
@@ -137,7 +137,7 @@ def self.create(lookup_invocation, config_path, owner)
           if parsed.is_a?(Hash)
             parsed
           else
-            Puppet.warning("#{config_path}: File exists but does not contain a valid YAML hash. Falling back to Hiera version 3 default config")
+            Puppet.warning(_("%{config_path}: File exists but does not contain a valid YAML hash. Falling back to Hiera version 3 default config") % { config_path: config_path })
```

Comment:
I think terms like YAML and JSON (acronyms) are fine to leave to the translators.

There's apparently some form of glossary of common Puppet terms that will be in use by translators and I expect generic technology terms will be covered as well.

I think what we want to call out is where there are possible cases of ambiguity, particularly those that would cause failure in a round-trip - ie a function name that could accidentally be translated to something else, and when printed in the non-english version might be intelligible, but will not work if the user actually uses the translated version as input to the system.